emily/cdk/lib/emily-stack.ts

```diff
@@ -279,14 +279,46 @@ export class EmilyStack extends cdk.Stack {
             deployOptions: { stageName: props.stageName },
         });
 
+        // Create a usage plan that will be used by the Signers. This will allow us to throttle
+        // the general API more than the signers.
+        const signerApiUsagePlanId: string = `SignerApiUsagePlan`;
+        const signerApiUsagePlan = api.addUsagePlan(signerApiUsagePlanId, {
+            name: EmilyStackUtils.getResourceName(signerApiUsagePlanId, props),
+            throttle: {
+                // These are very high limits. We can adjust them down as needed.
+                rateLimit: 100,
+                burstLimit: 200,
```

Comment:
Are these requests per second?

Reply:
Yes, we don't expect signers to get anywhere near this iirc.